src/main/java/org/example/ngevaticketmanagerspring/event/EventController.java

```diff
@@ -0,0 +1,82 @@
+package org.example.ngevaticketmanagerspring.event;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = "/api/event", produces = MediaType.APPLICATION_JSON_VALUE)
+public class EventController {
+
+    private final EventService service;
+
+    @PostMapping
+    public ResponseEntity<Event> createEvent(@Valid @RequestBody Event event) {
+        Event createdEvent = service.createEvent(event);
+        return ResponseEntity.status(HttpStatus.CREATED).body(createdEvent);
+    }
+
+    @PutMapping("{eventId}/{newQuota}")
+    public ResponseEntity<Object> updateQuota(@PathVariable Long eventId, @PathVariable int newQuota) {
+        try {
+            Event updatedEvent = service.updateQuota(eventId, newQuota);
+            return ResponseEntity.ok(updatedEvent);
+        } catch (RuntimeException e) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Error: " + e.getMessage());
+        }
+    }
```

Comment:
Ich würde den neuen Wert als Request Body mitgeben. Der Ansatz alles an den Path zu hängen geht spätestens dann kaputt, wenn du mehr als eine Sache ändern möchtest.
```suggestion
    @PutMapping("{eventId}")
    public ResponseEntity<Object> updateQuota(@PathVariable Long eventId, @RequestBody int newQuota) {
        try {
            Event updatedEvent = service.updateQuota(eventId, newQuota);
            return ResponseEntity.ok(updatedEvent);
        } catch (RuntimeException e) {
            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Error: " + e.getMessage());
        }
    }
```